sources/packages/api/src/services/education-program/education-program.service.models.ts

```diff
@@ -7,6 +7,7 @@ export interface SaveEducationProgram extends EducationProgramDto {
   id?: number;
   institutionId: number;
   approvalStatus: ApprovalStatus;
+  userId: number;
```

Comment:
Can you please elaborate in which scenario the _userId_ must come from the client? Is it not supposed to be consumed directly from the API?

Reply:
I got it now, thanks!